package.json

```diff
@@ -17,6 +17,7 @@
     "enhanced-resolve": "^4.1.0",
     "eslint-scope": "^4.0.0",
     "json-parse-better-errors": "^1.0.2",
+    "json-schema-to-typescript": "^6.0.1",
```

Comment:
Hi @sokra and @TheLarkInn,

shouldn't this package be added only into `devDependencies`?
Because of `json-schema-to-typescript` package being present now in dependencies, its own dependencies are also installed, examples:

```
 "@types/cli-color": "^0.3.29",
    "@types/json-schema": "^7.0.0",
    "@types/lodash": "^4.14.116",
    "@types/minimist": "^1.2.0",
    "@types/mz": "0.0.32",
    "@types/node": "^10.9.4",
    "@types/prettier": "^1.13.2",
```

I don't think it was the intent here, or am I missing something?

Reply:
I agree, it's only used in tooling. So, it should be in `devDependencies`.

Reply:
Yep you are right, fixing it.

Reply:
https://github.com/webpack/webpack/pull/8068